### DISCUSSION TITLE
阿里云盘这个refresh_token的you有效期一般是多久，每次修改后不久就失效了

### DISCUSSION THREAD

**Discussion by ihuangqiang** [messense/aliyundrive-webdav#237] [Q&A] [answered]
阿里云盘这个refresh_token的you有效期一般是多久，每次修改后不久就失效了，需要访问还要重新获取新的token更新才行

**Comment by messense** [comment, marked as answer]
https://github.com/messense/aliyundrive-webdav/discussions/110#discussioncomment-1580297

**Reply by messense** [reply]
善用搜索。
